File: system-test/java/src/main/java/com/optivem/shop/testkit/driver/port/shop/dtos/PublishCouponRequest.java
package com.optivem.shop.testkit.driver.port.shop.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PublishCouponRequest {
    private String code;
    private String discountRate;
    private String validFrom;
    private String validTo;
    private String usageLimit;
}

File: system-test/dotnet/Driver.Port/MyShop/Dtos/PublishCouponRequest.cs
namespace Driver.Port.MyShop.Dtos;

public class PublishCouponRequest
{
    public string? Code { get; set; }
    public string? DiscountRate { get; set; }
    public string? ValidFrom { get; set; }
    public string? ValidTo { get; set; }
    public string? UsageLimit { get; set; }
}
